.github/workflows/test.yml

```diff
@@ -218,14 +218,19 @@ jobs:
           
           # The runDirectoryTest.sh script expects the following env vars to be set
           export PLUGIN_DIR=$(nix build github:runtimeverification/blockchain-k-plugin/$PLUGIN_VERSION --no-link --json | $JQ -r '.[].outputs | to_entries[].value')
-          export SERVER=$(nix build .#kore-rpc-booster --no-link --json | $JQ -r '.[].outputs | to_entries[].value')/bin/kore-rpc-booster
+          KORE_RPC_BOOSTER=$(nix build .#kore-rpc-booster --no-link --json | $JQ -r '.[].outputs | to_entries[].value')/bin/kore-rpc-booster
+          BOOSTER_DEV=$(nix build .#hs-backend-booster:exe:booster-dev --no-link --json | $JQ -r '.[].outputs | to_entries[].value')/bin/booster-dev
           export CLIENT=$(nix build .#hs-backend-booster:exe:rpc-client --no-link --json | $JQ -r '.[].outputs | to_entries[].value')/bin/rpc-client
           
           cd test/rpc-integration
           for dir in $(ls -d test-*); do
               name=${dir##test-}
               echo "Running $name..."
-              ./runDirectoryTest.sh test-$name --time
+              if [ "$name" = "a-to-f" ]; then
+                  SERVER=BOOSTER_DEV ./runDirectoryTest.sh test-$name --time
+              else
+                  SERVER=KORE_RPC_BOOSTER ./runDirectoryTest.sh test-$name --time
+              fi
```

Comment:
I wanted to make sure we run the `a-to-f` tests with `booster-dev`, to ensure we are catching the vacuous branch in booster and not in the old backbend. (this happened during testing when the code in the booster wasn't yet working)